CMakeLists.txt

```diff
@@ -28,6 +28,7 @@ set(LOCAL_INIT_DIR "${CMAKE_BINARY_DIR}/local_init")
 option(BUILD_TESTS "Build the C++ tests" ON)
 
 # Look for packages early so we can exit if they're not found
+include(horace_Version)
```

Comment:
is there any reason for horaceVersion to be the first include?  Is there any ordering required on these imports?

Reply:
I don't think so, I'll move it down below the FindMatlab stuff since it uses variables defined in there - but more for our own sanity

Reply:
Actually - the Matlab version wasn't being added to the packages... whoops